### PR TITLE
[DOCFIX] Update Overview and use cases

### PR DESCRIPTION
**Pull request by apc999**
- Update Overview page
- Add a new use case page

### CODE REVIEW

File: docs/en/Overview.md [hunk 1]
```diff
@@ -7,43 +7,24 @@ group: Home
...
+
+Alluxio is the world’s first memory-speed virtual distributed storage
 system. It unifies data access and bridges computation frameworks and underlying storage systems.
 Applications only need to connect with Alluxio to access data stored in any underlying storage
 systems. Additionally, Alluxio’s memory-centric architecture enables data access at speeds that is
 orders of magnitude faster than existing solutions.
+It runs on commodity hardware, creating a shared data layer
+abstracting the files or objects in underlying persistent storage
+systems. Applications connect to Alluxio via a standard interface, accessing
+data from a single unified source.
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Overview.md]: fixed

File: docs/en/Overview.md [hunk 2]
```diff
@@ -55,48 +36,77 @@ The project is the storage layer of the Berkeley Data Analytics Stack
...
-``./bin/alluxio fs`` to interact with Alluxio, e.g. copy data in and out of the file system.
+<img src="{{site.baseurl}}{% link img/stack.png %}" width="800"/>
+
+## Benefits of Alluxio
+
+Alluxio helps overcome the obstacles to extracting value from data by making it
+simple to give applications access to whatever data is needed, regardless of
+format or location. The benefits of Alluxio Include:
+
+* **Memory-Speed I/O**: Alluxio
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Overview.md]: fixed

File: docs/en/Overview.md [hunk 3]
```diff
@@ -106,17 +116,9 @@ You can get the released versions of Alluxio from the
 [Project Downloads Page](http://alluxio.org/download). Each release comes with prebuilt
 binaries compatibile with various Hadoop versions. If you would like to build the project from the
 source code, check out the
-[Building From Master Branch Documentation](Building-Alluxio-From-Source.html). If you have any
+[Building From Master Branch Documentation]({{ site.baseurl }}{% link en/contributor/Building-Alluxio-From-Source.md %}). If you have any
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Overview.md]: correct

File: docs/en/Use-Cases.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+* Table of Contents
+{:toc}
+
+
+Here is a description of a few common use cases of Alluxio.
+
+## A Distributed Caching Layer for Data on Cloud or Object Storage
+
+In many companies or organizations, an increasingly popular architecture for
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Use-Cases.md]: done

File: docs/en/Use-Cases.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+adding extra ETL steps ahead. Deployed on the compute nodes on the satellite
+cluster and configured to connect to the main Hadoop cluster, an Alluxio service
+can serve like a local data proxy layer that provides the same namespace of the
+main HDFS data to the local compute jobs. In addition, Alluxio will
+transparently cache frequently accessed data to improve the locality and reduce
+network traffic of these jobs.
+
+See example use cases from [Tencent News](http://alluxio-com-site-prod.s3.amazonaws.com/resource/media/tencent-case-study-delivering-customized-news-to-over-100-million-montly-users).
+
+## Common Storage Access Service
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Use-Cases.md]: done

File: docs/en/Use-Cases.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+group: Home
+---
+
+* Table of Contents
+{:toc}
+
+
+Here is a description of a few common use cases of Alluxio.
+
+## A Distributed Caching Layer for Data on Cloud or Object Storage
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Use-Cases.md]: changed to object storage

**Comment by apc999** [comment]
@calvinjia  PTAL

File: docs/en/Use-Cases.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+- Lack of filesystem-level data caching so multiple jobs accessing the
+ same data set can benefit from caching the frequently accessed common
+ data.
+- Lack of node-level data locality on computation as data is always read
+ remotely and never node-local to the compute tasks.
+- Operations to list objects can be very slow especially when there is a massive
+ amount of objects in the bucket.
+- Operations to rename objects can be very slow and get weaker
+ consistency guarantees.
```
Comment (calvinjia) [inline docs/en/Use-Cases.md]: Can add throughput limitations (network bandwidth limit) and also security model is different from file systems.

File: docs/en/Use-Cases.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+execution or run ETL pipelines to preload data into satellite clusters prior to
+jobs execution.
+
+Alluxio can accelerate the remote data read from main Hadoop cluster without
+adding extra ETL steps ahead. Deployed on the compute nodes on the satellite
+cluster and configured to connect to the main Hadoop cluster, an Alluxio service
+can serve like a local data proxy layer that provides the same namespace of the
+main HDFS data to the local compute jobs. In addition, Alluxio will
+transparently cache frequently accessed data to improve the locality and reduce
+network traffic of these jobs.
```
Comment (calvinjia) [inline docs/en/Use-Cases.md]: Can add that Alluxio reduces load on the main cluster.

File: docs/en/Use-Cases.md (hunk 7 above)
Comment (apc999) [reply docs/en/Use-Cases.md]: added

File: docs/en/Use-Cases.md (hunk 8 above)
Comment (apc999) [reply docs/en/Use-Cases.md]: added

**Pull request merged**